<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lcPageSetupDialog</class>
 <widget class="QDialog" name="lcPageSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Page Setup</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Size</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="3">
       <widget class="QLineEdit" name="HeightEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="WidthEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Width:</string>
        </property>
        <property name="buddy">
         <cstring>WidthEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Height:</string>
        </property>
        <property name="buddy">
         <cstring>HeightEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Preset:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Units:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QComboBox" name="comboBox_2"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Margins</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Left:</string>
        </property>
        <property name="buddy">
         <cstring>LeftEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="LeftEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Right:</string>
        </property>
        <property name="buddy">
         <cstring>RightEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="RightEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Top:</string>
        </property>
        <property name="buddy">
         <cstring>TopEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="TopEdit"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Bottom:</string>
        </property>
        <property name="buddy">
         <cstring>BottomEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="BottomEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>WidthEdit</tabstop>
  <tabstop>HeightEdit</tabstop>
  <tabstop>comboBox_2</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>LeftEdit</tabstop>
  <tabstop>RightEdit</tabstop>
  <tabstop>TopEdit</tabstop>
  <tabstop>BottomEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>lcPageSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>lcPageSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
